data/categories.txt

```diff
@@ -6801,6 +6950,80 @@ vi:Quần vợt
 zh-Hans:网球
 zh-Hant:網球
 
+sport-volleyball
+en:Volleyball
+ar:كرة الطائرة
+be:Валейбол
+bg:Волейбол
+cs:Volejbal
+da:Volleyball
+de:Volleyball
+el:Πετοσφαίριση
+es:Voleibol
+eu:Boleibol
+fa:والیبال
+fi:Lentopallo
+fr:Volley-ball
+he:כדורעף
+hu:Röplabda
+id:Bola voli
+it:Pallavolo
+ja:バレーボール
+ko:배구
+mr:व्हॉलीबॉल
+nb:Volleyball
+nl:Volleyball
+pl:Piłka siatkowa
+pt:Voleibol
+ro:Volei
+ru:Волейбол
+sk:Volejbal
+sv:Volleyboll
+sw:Voliboli
+th:วอลเลย์บอล
+tr:Voleybol
+uk:Волейбол
+vi:Bóng chuyền
+zh-Hans:排球
+zh-Hant:排球
+
+sport-9pin|sport-10pin
+en:Nine-pin Bowling|Bowling
```

Comment:
Let's make it simple "Bowling", I doubt that someone will use Nine-pin in the search.